### PR TITLE
Add @bryanvdelft

### PR DESCRIPTION
**Pull request by bryanvdelft**

### CODE REVIEW

File: site/class-2-debug/bryanvdelft/index.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  margin: 0;
+}
+
+svg {
+  width: 100vw;
+  height: 100vh;
+}
+
+.axis path {
+  fill: no;
```
Comment (wooorm) [inline site/class-2-debug/bryanvdelft/index.css]: no -> none

File: site/class-2-debug/bryanvdelft/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<!--Based on https://bl.ocks.org/mbostock/3887118 by Mike Bostock-->
+<meta charset=utf8>
+<!--Prevent favicon error from showing up-->
+<link rel="shortcut icon" type=image/x-icon href=data:image/x-icon;,>
+<title>@bryanvdelft</title>
+<meta content=width=device-width,initial-scale=1 name=viewport>
+<link rel=stylesheet href=../index.css>
```
Comment (wooorm) [inline site/class-2-debug/bryanvdelft/index.html]: ../index.css -> index.css

File: site/class-2-debug/bryanvdelft/index.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+<body>
+    <svg width="1000" height="1000"></svg>
+</body>
+
+<script src="https://d3js.org/d3.v4.min.js"></script>
+<script src=index.js></script>
```
Comment (wooorm) [inline site/class-2-debug/bryanvdelft/index.html]: These scripts should go in the body.

**Comment by bryanvdelft** [comment]
Aangepast!

**Pull request merged**